source/connect-zbt1/index.html

```diff
@@ -1,14 +1,14 @@
 ---
 layout: landingpage
-title: "Home Assistant SkyConnect"
+title: "Home Assistant Connect ZBT-1"
 description: "Best way to add Zigbee and Thread to Home Assistant"
 date: 2022-10-05
-tagline: Home Assistant SkyConnect
+tagline: Home Assistant Connect ZBT-1
 tagline_wide: false
-og_image: /images/skyconnect/skyconnect-cover.png
-hero_image: /images/skyconnect/skyconnect_horizontal.png
+og_image: /images/connectzbt1/connectzbt1-cover.jpg
```

Comment:
Probably makes sense to remove the skyconnect folder under images

Reply:
Will do that in a separate pR!